web/modules/custom/farm_nfa/src/Form/ForestPlanGfwForm.php

```diff
@@ -27,9 +27,13 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     // Set the form title.
     $form['#title'] = $this->t('GFW');
 
-    $form['placeholder'] = [
-      '#type' => 'markup',
-      '#markup' => '(GFW form placeholder)',
+    $form['gfw_map_placeholder'] = [
```

Comment:
A placeholder is a text that is there until is replaced by something else, we've now replaced it by a map, so "map" or "gfw_map" makes more sense

Reply:
got it !